static/app/gettingStartedDocs/python/aiohttp.tsx

```diff
@@ -94,56 +81,47 @@ const onboarding: OnboardingConfig = {
           language: 'python',
           code: `
 ${getSdkSetupSnippet(params)}
-async def hello(request):
-   return web.Response(text="Hello, world")
+###the following lines of code
+###are for testing
+async def debug(request):
+   1/0  # raises an error
+   return web.Response(text="Debug with Sentry")
 
 app = web.Application()
-app.add_routes([web.get('/', hello)])
+###this one too
+app.add_routes([web.get('/sentry-debug', sentry-debug)])
 
 web.run_app(app)
         `,
         },
       ],
     },
   ],
-  verify: (params: Params) => [
+  verify: () => [
     {
       type: StepType.VERIFY,
-      description: t(
-        'You can easily verify your Sentry installation by creating a route that triggers an error:'
-      ),
-      configurations: [
+      description: tct(
+        'You can easily verify your Sentry installation by creating a route that triggers an error. The above snippet includes an endpoint [codeDebug:GET /sentry-debug].',
         {
-          language: 'python',
-
-          code: `
-          ${getSdkSetupSnippet(params)}
-async def hello(request):
-  1/0  # raises an error
-  return web.Response(text="Hello, world")
-
-app = web.Application()
-app.add_routes([web.get('/', hello)])
-
-web.run_app(app)`,
-        },
-      ],
+          codeDebug: <code />,
+        }
+      ),
       additionalInfo: (
         <span>
           <p>
             {tct(
-              `When you point your browser to [localhostLInk:http://localhost:8080/] a transaction in the Performance section of Sentry will be created.`,
+              `When you point your browser to [link:http://localhost:8080/sentry-debug/] an error with a trace will be created. So you can explore errors and tracing portions of Sentry.`,
```

Comment:
wouldn't it be "So you can explore errors and tracing portions of _your app in_ Sentry"?